### PR TITLE
Major rewrites

### PR DESCRIPTION
**Pull request by nothingneko**

### CODE REVIEW

**Comment by ghost** [comment]
whoops miss click

File: src/pages/services.astro [hunk 1]
```diff
@@ -119,6 +121,41 @@ const seo = {
...
+          <div
+            class="p-4 bg-neutral-100 dark:bg-neutral-800 hover:shadow-md transition-all rounded-2xl"
+          >
+            <p class="font-semibold text-2xl mb-3 dark:opacity-[92%]">
+              UI/UX Design
+            </p>
+            <p class="dark:opacity-[92%]">
+              Improve your tools with our UI/UX expertise. We provide your team
+              with knowledgable assitance in creating intuitive designs with
+              fluid interactions, we
```
Comment (madonuko) [inline src/pages/services.astro]: wha

File: src/pages/projects.astro [hunk 2]
```diff
@@ -75,10 +75,10 @@ const seo = {
             </h1>
 
             <p class="dark:opacity-[92%]">
-              Ultramarine Linux is a Fedora-based Linux distribution designed to
```
Comment (lleyton) [inline src/pages/projects.astro:78]: Why remove all references to Fedora?

Comment (nothingneko) [reply src/pages/projects.astro:78]: Felt odd to point out our upstream in a brief summary, makes more sense on the full page than here - Also worried about WiiUitis

**Review by lleyton — APPROVED** [review]

**Pull request merged**